test/integration_test.rb

```diff
@@ -22,7 +70,7 @@ def run_vernier(options, *argv)
       env = options.map do |k, v|
         ["VERNIER_#{k.to_s.upcase}", v.to_s]
       end.to_h
-      result = system(env, RbConfig.ruby, "-I", LIB_DIR, "-r", "vernier/autorun", *argv, exception: true)
+      system(env, RbConfig.ruby, "-I", LIB_DIR, "-r", "vernier/autorun", *argv, exception: true, out: '/dev/null', err: '/dev/null')
```

Comment:
the output here was spammy, added redirections to /dev/null to keep the tests quiet.